lib/tools/bundletool.js

```diff
@@ -0,0 +1,114 @@
+import { exec } from 'teen_process';
+import log from '../logger.js';
+import path from 'path';
+import _ from 'lodash';
+import {getJavaForOs} from "../helpers";
+
+let bundletool = {};
+
+/**
+ * Check bundletool version
+ *
+ * @return {string} Version of bundletool. i.e., BundleTool 0.6.0
+ * @throws {Error} If get version fails
+ */
+bundletool.version = async function () {
+  try {
+    return await exec(getJavaForOs(), [
+      '-jar', path.resolve(this.helperJarPath, 'bundletool.jar'), 'version'
+    ]);
+  } catch (e) {
+    throw new Error(`Could not get bundle tool version. Original error ${e.message}`);
+  }
+};
+
+/**
+ * Install apks
+ *
+ * @param {string} apks - The full path to the local apks file.
+ * @param {string} deviceId - The device serial to install the apks to.
+ * @param {[string]} modules - List of modules to be installed (defaults to all of them).
+ *                           Note that the dependent modules will also be installed.
+ *                           Ignored if the device receives a standalone APK.
+ * @return {string} Command stdout
+ * @throws {Error} If installation fails. e.g. INSTALL_PARSE_FAILED_NO_CERTIFICATES
+ */
+bundletool.installApks = async function (apks, deviceId, modules = []) {
+  // TODO: Append install options after bundletool implement it.
+  // https://github.com/google/bundletool/blob/f855ea639a02216780b2813ce29bd6e927ad4503/src/main/java/com/android/tools/build/bundletool/device/DdmlibDevice.java#L89-L107
+  let installArgs = [
+    '-jar', path.resolve(this.helperJarPath, 'bundletool.jar'), 'install-apks',
+    '--apks', apks, '--device-id', deviceId
+  ];
+
+  if (!_.isEmpty(modules)) {
+    installArgs.push('--modules', modules.join(','));
+  }
+
+  log.info(`Install apks with: ${installArgs}`);
+  try {
+    return await exec(getJavaForOs(), installArgs);
+  } catch (e) {
+    throw new Error(`Failed to install apks. Original error ${e.message}`);
+  }
+};
+
+/**
+ * Check build version
+ *
+ * @param {string} bundle - The full path to the local `aab` file.
+ * @param {string} deviceId - The device serial to install the apks to.
+ * @param {string} output - List of modules to be installed (defaults to all of them).
+ *                           Note that the dependent modules will also be installed.
+ *                           Ignored if the device receives a standalone APK.
+ * @param {object} buildApksOpts - Options
```

Comment:
there is `typedef` doctring syntax for defining such things. Please check the other modules for more examples